components/project-card/meta/issues-and-tags.jsx

```diff
@@ -20,21 +21,22 @@ const IssuesAndTags = props => {
   let tags = props.tags.map(tag => {
     return (
       <Link
-        to={`/tags/${encodeURIComponent(tag)}`}
-        className="btn btn-xs btn-tag"
-        key={tag}
+      to={`/tags/${encodeURIComponent(tag)}`}
+      key={tag}
       >
-        {tag}
+        #{tag}
       </Link>
     );
   });
 
   let classnames = classNames(`issues-and-tags`, props.className);
+  let issues_and_tags = issues.concat(tags);
+
+  issues_and_tags = issues_and_tags.reduce((accu, curr) => [accu, ", ", curr]);
```

Comment:
@youriwims do you wanna change this to something easier to understand?

Reply:
Nope—I'm good with this now 😸